{"name": "test_sub[字符串减正数异常用例]", "status": "passed", "parameters": [{"name": "a", "value": "'a'"}, {"name": "b", "value": "11"}, {"name": "expect", "value": "'TypeError'"}], "start": 1603368605989, "stop": 1603368605989, "uuid": "bcf863f9-b16c-40d0-bd7b-b3e2802ca1f0", "historyId": "dfe584ad1a46125c943165ea71814b18", "testCaseId": "69937ce96b995c8a55ed35f093129d52", "fullName": "test_calc2.TestCalcTaskOne#test_sub", "labels": [{"name": "tag", "value": "sub"}, {"name": "tag", "value": "run(order=2)"}, {"name": "suite", "value": "test_calc2"}, {"name": "subSuite", "value": "TestCalcTaskOne"}, {"name": "host", "value": "lingz-mac.lan"}, {"name": "thread", "value": "50460-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc2"}]}